scripts/lint-skills-wiring.sh

```
@@ -0,0 +1,64 @@
+    parent="$(dirname "$dir")"
+    if [[ "$parent" != "skills" && -f "$parent/SKILL.md" ]]; then
+      continue
+    fi
+    printf '%s\n' "$dir"
+  done
+)"
+[[ -n "$skill_dirs" ]] || { err "no skills found under skills/"; exit 1; }
+
+names="$(printf '%s\n' "$skill_dirs" | while IFS= read -r d; do basename "$d"; done | sort)"
+dups="$(printf '%s\n' "$names" | uniq -d)"
+[[ -z "$dups" ]] || err "duplicate skill names across groups: $(printf '%s ' $dups)"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)